run/luks2john.py

```diff
@@ -49,7 +50,7 @@
 
 def af_sectors(blocksize, blocknumbers):
     af_size = blocksize * blocknumbers
-    af_size = (af_size + 511) / 512
+    af_size = (af_size + 511) // 512
```

Comment:
Are you sure about this change? The `+ 511` will ensure rounding upwards. Won't the `//` make it round upwards once again?

Reply:
``` Python
af_size = blocksize * blocknumbers                                           
af_size = (af_size + 511) // 512                                             
af_size *= 512 
```

I tested this change experimentally and it works fine.

Under Python 3, `(af_size + 511) / 512` produces a float quantity which doesn't produce correct results later on.

Reply:
Wouldn't just `af_size // 512` produce a correct rounded-up figure?

Reply:
The `//` operator doesn't do any rounding-up.

```
$ bpython
>>> 511 // 512 
0
>>> 512 // 512
1
>>> 513 // 512
1
```

Reply:
Oh, it's floor division, not ceiling 😆 .  I had it backwards, just ignore me!

Reply:
No worries :-)